ui/tests/acceptance/secrets/backend/cubbyhole/secret-test.js

```diff
@@ -46,6 +46,7 @@ module('Acceptance | secrets/cubbyhole/create', function (hooks) {
       'vault.cluster.secrets.backend.show',
       'redirects to the show page'
     );
+    assert.dom('[ data-test-created-time]').hasText('', 'it does not render created time if blank');
```

Comment:
I don't think this affects the test, but there is an extra space `[ data-test-created-time]`

Reply:
thanks! I copied this directly from the original test - so I'll delete it there too!